lib/logger.js

```diff
@@ -96,8 +96,9 @@ logger.log = function (...args) {
 logger.banner = function () {
 	var info = appc.pkginfo.package(module, 'version', 'about');
 	if (bannerEnabled) {
-		logger.log(info.about.name.cyan.bold + ', CLI version ' + info.version + (logger.activeSdk ? ', Titanium SDK version ' + logger.activeSdk : '') + '\n' + info.about.copyright);
-		logger.log('\n' + __('Please report bugs to %s', 'https://github.com/tidev/titanium_mobile/issues'.cyan) + '\n');
+		logger.log(info.about.name.cyan.bold + ', CLI v' + info.version + (logger.activeSdk ? ', Titanium SDK v' + logger.activeSdk : '') + '\n' + info.about.copyright);
```

Comment:
I think this could all be condensed further down to:

```
Titanium CLI v5.4.1, SDK v11.0.0, https://titaniumsdk.com
Want to help? https://tidev.io/donate or https://tidev.io/contribute

Usage: titanium <command> [options]

Commands:
  <snip>
```

1. I think everybody knows what a CLI is at this point.
2. I really don't think we need to print the copyright. The license file is distributed with the CLI and SDK. I'm not a lawyer, but it seems most CLI's do not display copyright info in the banner.
3. Put the titaniumsdk.com URL in the top line.
4. Second line, add Jason's donate link and a link to contribute. This is essentially what Mozilla Thunderbird does.
5. We don't need link a link to report a bug. This is legacy because Appc used JIRA. Now everything is on GitHub and it's easy to find.

That's my 2 cents.

Reply:
https://tidev.io/donate has to be made before release. Can you do that first or should I on the "old" page? Dunno how far the new website is

Reply:
I'm literally working on it right now. New site is not done. Remember, I have no design, no content. I'm just trying to get the pieces together including the blog.

Reply:
<img width="1361" alt="Bildschirmfoto 2022-05-19 um 07 14 40" src="https://user-images.githubusercontent.com/1126641/169213858-9f75857f-5914-4216-bca8-ca842f1a7fed.png">